### DISCUSSION TITLE
Where is binary release.. moreover that one for Windows

### DISCUSSION THREAD

**Discussion by abdulbadii** [fosskers/aura#839] [Q&A]
Where is a binary release, especially one that for Msys2/Mingw64 on Windows ?
one please generous enough to build and contribute it?

**Comment by fosskers** [comment]
Hi, I've been out of town but will be producing better release packages soon.

**Reply by fosskers** [reply]
I have just released `aura-git` that targets the new `4.0.0`: https://aur.archlinux.org/packages/aura-git
